src/app/app.ts

```diff
@@ -72,7 +73,7 @@ class AppImpl implements App {
 
         const id = zdTicket.id;
         const subject = zdTicket.subject;
-        const message = `${zdUser.name} created ticket [#${id}](${host}/agent/tickets/${id}) [${subject}]`;
+        const message = `${zdUser.name} created ticket [#${id}](${host}/agent/tickets/${id})  **Subject:** \`${subject}\``;
```

Comment:
Do we need the `**Subject:**` here? Could we instead just put a colon after the id?